File: du-backend/src/main/java/com/dwin/du/api/request/AddEditPersonRequest.java
package com.dwin.du.api.request;
import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class AddEditPersonRequest {
    public String name;
    public String surname;

    public AddEditPersonRequest(String name, String surname) {
        this.name = name;
        this.surname = surname;
    }
}

File: divvyup/DivvyUp.Shared/RequestDto/AddEditPersonRequest.cs
namespace DivvyUp_Shared.RequestDto
{
    public class AddEditPersonRequest
    {
        public string Name { get; set; }
        public string Surname { get; set; }

        public AddEditPersonRequest()
        {
            Name = string.Empty;
            Surname = string.Empty;
        }

        public AddEditPersonRequest(string name, string surname)
        {
            Name = name;
            Surname = surname;
        }
    }
}